roles/cs.nginx-https-termination/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-https_termination_varnish_upstream: "127.0.0.1:{{ varnish_port | default(80) }}"
+https_termination_upstream: "127.0.0.1:{{ varnish_port | default(80) }}"
```

Comment:
I've changed the name because the upstream is not always varnish (in case of raccoon it will be the application ghost). This var is not used anywhere, it's always the default so no regressions are possible.